packages/aws-cdk-lib/aws-ecs/lib/amis.ts

```diff
@@ -143,6 +148,7 @@ export class EcsOptimizedAmi implements ec2.IMachineImage {
       + (this.windowsVersion ? `Windows_Server-${this.windowsVersion}-English-Full-ECS_Optimized/` : '')
       + (this.hwType === AmiHardwareType.GPU ? 'gpu/' : '')
       + (this.hwType === AmiHardwareType.ARM ? 'arm64/' : '')
+      + (this.hwType === AmiHardwareType.NEURON ? 'inf/' : '')
```

Comment:
Should we test this change in the unit tests or is there no way to do that?

Reply:
Added a unit test to check the template has the correct SSM parameter